xonsh/tools.py

```diff
@@ -890,9 +890,8 @@ def escape_windows_cmd_string(s):
     The escaping is based on details here and empirical testing:
     http://www.robvanderwoude.com/escapechars.php
     """
-    for c in '()%!^<>&|"':
+    for c in '^()%!<>&|"':
         s = s.replace(c, '^' + c)
-    s = s.replace('/?', '/.')
```

Comment:
@rbrewer123 I removed this from the `escape_windows_cmd_string()`. I think it is something you once added, but I can't see what the purpose of it is.